epigenetics.yaml.lock

```diff
@@ -42,10 +42,13 @@ tools:
 - name: chipeakanno_annopeaks
   owner: kpbioteam
   revisions:
+  - 104882a2de00
   - 6f52c0dc56d4
   tool_panel_section_label: Test Tools
 - name: ewastools
   owner: kpbioteam
   revisions:
+  - 369fef559cfc
```

Comment:
@kpbioteam are both revisions here needed or just the latest one?

Reply:
@bgruening just the latest one